react-bootstrap-table/react-bootstrap-table.d.ts

```diff
@@ -520,11 +558,13 @@ declare module "react-bootstrap-table" {
 		options?: any;
 	}
 	
+	export type FilterType = 'TextFilter' | 'RegexFilter' | 'SelectFilter' | 'NumberFilter' | 'DateFilter' | 'CustomFilter';
```

Comment:
As I understand the code, `'CustomFilter'` is the way to add a [custom filter](https://github.com/AllenFang/react-bootstrap-table/blob/80bad67aa31a1471e3ffa06b9a98baa4382c086f/src/Const.js#L28-L35).

Hmm, but maybe the `FilterType` should be a `string`. The documentation seems to suggest that any string is allowed here.